src/uta/models.py

```diff
@@ -210,8 +210,8 @@ class ExonAln(Base):
     cigar = sa.Column(sa.Text, nullable=False)
     added = sa.Column(
         sa.DateTime, default=datetime.datetime.now(), nullable=False)
-    tx_aseq = sa.Column(sa.Text, nullable=False)
-    alt_aseq = sa.Column(sa.Text, nullable=False)
+    tx_aseq = sa.Column(sa.Text, nullable=True)
```

Comment:
These defaults had to be changed to be compatible with the data in the `exon_aln` table.